src/components/registration/FamilyDetailsSidebar.jsx

```diff
@@ -31,20 +41,21 @@ function FamilyDetailsSidebar({ isOpen, rowData, handleClose }) {
       onClose={handleClose}
     >
       <h3>
-        Hey {firstName} {lastName}!
+        Hey {family.parent.first_name} {family.parent.last_name}!
       </h3>
     </Drawer>
   );
 }
 
 FamilyDetailsSidebar.defaultProps = {
   isOpen: false,
+  familyId: null,
   handleClose: () => {},
 };
 
 FamilyDetailsSidebar.propTypes = {
   isOpen: PropTypes.bool,
-  rowData: PropTypes.arrayOf(PropTypes.string).isRequired,
+  familyId: PropTypes.number,
```

Comment:
hmm, it might not make sense to open this sidebar if the family id is null - can we make this required?
if so, we can also remove the if statement on line 29